bindings/python/scanners.i

```diff
@@ -14,8 +13,16 @@ class BaseScanner {
 }
 
 
+%typemap(out) PtrdiffOrNone {
+    if ($1.second == NULL || $1.first == NULL)
```

Comment:
PIRE сейчас компилируется без указания стандарта

15:21, пт, 05.08.2016 sergey-v-galtsev <notifications@github.com>:

> In bindings/python/scanners.i
> <https://github.com/moskupols/pire/pull/4#discussion_r73683454>:
>
> > @@ -14,8 +13,16 @@ class BaseScanner {
> >  }
> >
> >
> > +%typemap(out) PtrdiffOrNone {
> > +    if ($1.second == NULL || $1.first == NULL)
>
> ИМХО, s/NULL/nullptr/g если PIRE сейчас компилируется как C++11.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/moskupols/pire/pull/4/files/e3b4f18922cef0dc890ea1957c77f263048cc103..76f58fb162e59a8c91e244d1ff5e5e54b158bc85#r73683454>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF8wxJyH2NDbIgGv1zo44yX0cbkSCb3Xks5qcyrKgaJpZM4JXXfn>
> .
>